src/Record/Record.Model/RecordBuilder.cs

```diff
@@ -349,17 +350,17 @@ public Record Build()
     {
         if (_storage.Id == null) throw new RecordException("Record needs ID.");
 
-        var metadataGraph = CreateMetadataGraph();
+        var metaDataGraph = CreateMetadataGraph();
```

Comment:
Metadata is a single word, it is not split. "Meta data" is a description of data being meta. "Metadata" is a word for such 
data.

https://en.wikipedia.org/wiki/Metadata
```suggestion
        var metadataGraph = CreateMetadataGraph();
```